### DISCUSSION TITLE
Would this work on Mac?

### DISCUSSION THREAD

**Discussion by KiiDbeamO** [QuestCraftPlusPlus/QuestCraft#194] [Q&A]
Dose same apply for mac, i don't see no extract

**Comment by wellsilver** [comment]
If sidequest works with mac then yes

and if you are (meh) amount of tech savvy you could do it without sidequest and use ABD

**Reply by Tman564ws1** [reply]
SideQuest works on Max so you can install it. And also you can even Port your mods. As long as they're on the quest craft spreadsheet.

**Comment by SmashHits** [comment]
i would also suggest getting amaze file manager installed onto your quest so you can launch apks from other sources natively
